### ISSUE TITLE
Error on Node 10.15

### ISSUE THREAD

**Issue by richorama** [JeanSebTr/file-watcher#1]
I'm trying the code sample on the readme, and I get this error on Node 10.15.

```
fs.js:670
  binding.stat(pathModule._makeLong(path), callback);
          ^
TypeError: path must be a string
    at Object.fs.stat (fs.js:670:11)
    at EventEmitter.Watcher.watch (/home/rich/Code/myproject/node_modules/file-watcher/index.js:20:8)
```

Any ideas?

**Comment by richorama** [comment]
It looks like the `watcher.watch` function expects a path and a callback.

**Comment by JeanSebTr** [comment]
Thanks that's fixed in v0.0.2 which is on npm now :)

You can use the root option of the constructor instead of using watch(dir)
The callback is also optional.
